tests/integration/member_test.go

```diff
@@ -115,3 +119,52 @@ func TestSnapshotAndRestartMember(t *testing.T) {
 		}
 	}
 }
+
+func TestRemoveMember(t *testing.T) {
+	integration.BeforeTest(t)
+	c := integration.NewCluster(t, &integration.ClusterConfig{Size: 3, UseBridge: true, BackendBatchInterval: 1000 * time.Second})
+	defer c.Terminate(t)
+	time.Sleep(etcdserver.HealthInterval)
```

Comment:
```suggestion
	c.WaitLeader(t)
```

Reply:
This is not about waiting for leader, membership changes additionally require cluster to be stable for `etcdserver.HealthInterval`. Please leave a comment to explain this.